lib/modules/showImages.js

```diff
@@ -1345,35 +1345,30 @@ addModule('showImages', (module, moduleID) => {
 		$(header).safeHtml(imageLink.imageTitle);
 		imgDiv.appendChild(header);
 
-		const template = await RESTemplates.load('GfycatUI');
```

Comment:
This is for .gifs that get converted to gfycat videos on-the-fly, IIRC.

If you're feeling particularly adventurous, you could try to run them through `hosts/gfycat.js` after they're converted, but `showImages` is a bit of a mess - it probably won't be easy or clean to do.